Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -10631,6 +10637,8 @@
 		4471710C205AF945000A116E /* MediaQueryParserContext.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = MediaQueryParserContext.h; sourceTree = "<group>"; };
 		447958021643B47B001E0A7F /* ParsedContentType.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = ParsedContentType.cpp; sourceTree = "<group>"; };
 		447958031643B47B001E0A7F /* ParsedContentType.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = ParsedContentType.h; sourceTree = "<group>"; };
+		447B137C2CB6B80C00F355CB /* JSCaretPositionFromPointOptions.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; name = JSCaretPositionFromPointOptions.h; path = "/Users/rbuis/Library/Developer/Library/Developer/Xcode/DerivedData/WebKit-dvwybpzhkzbzyseogofxedcfxxir/Build/Products/Debug/DerivedSources/WebCore/JSCaretPositionFromPointOptions.h"; sourceTree = "<absolute>"; };
+		447B137D2CB6B80C00F355CB /* JSCaretPositionFromPointOptions.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; name = JSCaretPositionFromPointOptions.cpp; path = "/Users/rbuis/Library/Developer/Library/Developer/Xcode/DerivedData/WebKit-dvwybpzhkzbzyseogofxedcfxxir/Build/Products/Debug/DerivedSources/WebCore/JSCaretPositionFromPointOptions.cpp"; sourceTree = "<absolute>"; };
```

Comment:
Can’t check this in with paths that are absolute and include your user name! Needs to be changed to be group-relative.

Reply:
Yes, that was sloppy, should have checked. Fixed.